### PR TITLE
Added warnings error category and filtered errors with "data criteria…

### PR DESCRIPTION
**Pull request by lmd59**
… outside" to this tab

### CODE REVIEW

File: app/models/qrda_file.rb [hunk 1]
```diff
@@ -34,13 +34,18 @@ def process
...
-    self.validation_errors = { qrda: [], reporting: [], submission: [], ungrouped: [] }
+    self.validation_errors = { qrda: [], reporting: [], submission: [], warning: [], ungrouped: [] }
     validators.each do |v|
       errs = v.validate(content, file_name: @filename)
       errs.each do |e|
         e.validator ||= v.class.name
-        self.validation_errors[ validator_category(e.validator) ] << e.instance_values
-
+        #make cat1 validation error types a warning for data criteria outside the measure definition
+        byebug
```
Comment (dczulada) [inline app/models/qrda_file.rb]: Remove debug breakpoint

**Comment by okeefm** [comment]
How does the CVU handle this warning data on the front-end? Does it just work, or is that out-of-scope for this pull request?

File: app/models/qrda_file.rb (hunk 1 above)
Comment (lmd59) [inline app/models/qrda_file.rb]: ... sorry 'bout that.

**Comment by lmd59** [comment]
The warning data is in a new "Warnings" tab (instead of in "Reporting Errors" as it was previously)
screen shot 2016-10-26 at 8 43 56 am

**Comment by okeefm** [comment]
Cool!

**Pull request merged**